code/modules/research/xenobiology/vatgrowing/vatgrower.dm

```diff
@@ -4,6 +4,10 @@
 	desc = "Tastes just like the chef's soup."
 	icon_state = "growing_vat"
 	buffer = 300
+	///category for plumbing RCD
+	category="Synthesizers"
```

Comment:
Looks good! Could you just format these like the following so it better follows [our styleguide](https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#spacing )?
```suggestion
	category = "Synthesizers"
```

Reply:
I keep forgetting to put spaces  ><